pkg/controller/certificatesigningrequest/common.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package certificatesigningrequest
+
+import (
+	"crypto/x509"
+	"fmt"
+	"sort"
+	"time"
+
+	certificates "k8s.io/api/certificates/v1"
+	sautil "k8s.io/apiserver/pkg/authentication/serviceaccount"
+)
+
+const (
+	// Set resyncPeriod to 0 to disable resyncing.
+	resyncPeriod time.Duration = 0
+	// How long to wait before retrying the processing of a CertificateSigningRequest change.
+	minRetryDelay = 5 * time.Second
+	maxRetryDelay = 300 * time.Second
+	// Default number of workers processing a CertificateSigningRequest change.
+	defaultWorkers = 4
+	// CertificateBlockType is a possible value for pem.Block.Type.
+	certificateBlockType = "CERTIFICATE"
+	antreaOrganization   = "antrea.io"
+)
+
+const (
+	// antreaIPsecCSRSignerName is the signer name for signing IPsec certificates
+	antreaIPsecCSRSignerName = "antrea.io/ipsec"
```

Comment:
@tnqn @antoninbas @jianjuns Do you think `antrea.io/ipsec` is a good signer name for IPsec certificates?
Some well-known signer names provided by K8s:
- kubernetes.io/kube-apiserver-client
- kubernetes.io/kube-apiserver-client-kubelet
- kubernetes.io/kubelet-serving

Reply:
I see requester and usage in above examples, then how about `antrea.io/antrea-agent-ipsec-tunnel`?